examples/jwt_auth.rs

```diff
@@ -0,0 +1,36 @@
+use std::env::var;
+
+use dotenv::dotenv;
+use trino_rust_client::auth::Auth;
+use trino_rust_client::ssl::Ssl;
+use trino_rust_client::{ClientBuilder, Row};
+
+#[tokio::main]
+async fn main() {
+    dotenv().ok();
+
+    let user = var("Bearer xxxxx").unwrap();
+    let access_token = var("Bearer xxxxx").unwrap();
```

Comment:
I'm not sure to understand what you're doing here, could you explain?

Reply:
my bad, updated.